update.sh

```diff
@@ -0,0 +1,162 @@
+#!/bin/bash
+
+set -euo pipefail
+
+msg () {
+	# shellcheck disable=SC2048,2086 # we want to expand words to allow -n flag
+	1>&2 echo $*
+}
+
+# Set global vars ARCHIVE_FILE ARCHIVE_ROOT
+get_archive_vars () {
+	# choose the archive file with the largest version number, use this to update from
+	ARCHIVE_FILE="$(find . -name 'govuk-prototype-kit*.zip' -exec basename '{}' ';' | sort -V | tail -n1)"
+	ARCHIVE_ROOT="${ARCHIVE_FILE//.zip}"
+}
+
+# Hide update folder from git
+update_gitignore () {
+	cd update
+
+	if [[ -f .gitignore ]]; then
+		cp .gitignore .gitignore.bak
+	fi
+
+	echo '*' > .gitignore
+
+	cd ..
+}
+
+# Check whether it is safe for the script to run
+check () {
+	if ! grep -q '"govuk-prototype-kit"' package.json 2> /dev/null; then
+		msg 'ERROR you must run update.sh in a folder containing a GOV.UK Prototype Kit installation'
+		msg 'Exiting'
+		exit 1
+	fi
+}
+
+# Lay the ground work
+prepare () {
+	# Clean any old update folder, unless asked not to
```

Comment:
I wonder if we could expand the comments here to explain that we're removing old update folders (in case the script has been run before)?